deb/common/rules

```diff
@@ -63,5 +63,13 @@ override_dh_install:
 	# TODO Can we do this from within our container?
 	dh_apparmor --profile-name=docker-ce -pdocker-ce
 
+override_dh_installman:
+	# based on ubuntu's "unminimize" script
+	if [ "$$(dpkg-divert --truename /usr/bin/man)" = "/usr/bin/man.REAL" ]; then \
+		rm -f /usr/bin/man \
+		&& dpkg-divert --quiet --remove --rename /usr/bin/man; \
```

Comment:
took this from the Ubuntu script, but perhaps a simple `mv /usr/bin/man.REAL /usr/bin/man` would be enough

Reply:
I'm just wondering if this is actually the correct to way to do this, like should our package be the one to "uniminimize" man pages or should that be handled by ubuntu's own unminimizing scripts.

Like the OS had a reason to minimize so I don't think that we should be the one's to go through and unminimize on behalf of the OS.

Reply:
this would only replace the `man` script with the actual `man` binary; and only during our packaging. Not sure what other options we have; we should report a bug upstream I guess (so that the `dh_installman` script fails if it's not an actual `man`, but the upstream is Debian, so I guess they would point to Ubuntu

Reply:
I have filed a bug report against Ubuntu's version of the debhelper package: <https://bugs.launchpad.net/ubuntu/+source/debhelper/+bug/1833124>